packages/mapviewer/tests/cypress/tests-e2e/importToolFile.cy.js

```diff
@@ -563,61 +583,98 @@ describe('The Import File Tool', () => {
             '@headInvalidKmlFile',
             '@getInvalidKmlFile',
             '@headUnreachableKmlFile',
             '@getUnreachableKmlFile',
             '@headOutOfBoundKmlFile',
             '@getOutOfBoundKmlFile',
-        ])
-        cy.readStoreValue('state.layers.activeLayers').should('have.length', 3)
+            '@headValidKmlFileWrongContentType',
+            '@getValidKmlFileWrongContentType',
+        ]
+        cy.wait(kmlRequests)
+
+        // Validate store and visible layers
+        cy.readStoreValue('state.layers.activeLayers').should('have.length', 4)
         cy.get('[data-cy="menu-section-active-layers"]')
             .should('be.visible')
             .children()
-            .should('have.length', 3)
+            .should('have.length', 4)
             .each(($layer, index) => {
                 cy.wrap($layer)
                     .find('[data-cy="menu-external-disclaimer-icon-cloud"]')
                     .should('be.visible')
-                cy.wrap($layer)
-                    .find('[data-cy^="button-has-error"]')
-                    .should('be.visible')
-                    .trigger('mouseover')
-                if (index === 0) {
-                    cy.get(`[data-cy^="floating-button-has-error-${onlineUrlNotReachable}"]`)
+
+                const errorData = [
+                    {
+                        url: validOnlineUrlWithInvalidContentType,
+                        shouldHaveError: false,
+                    },
+                    {
+                        url: onlineUrlNotReachable,
+                        shouldHaveError: true,
+                        errorMessage: 'file not accessible',
+                    },
+                    {
+                        url: invalidFileOnlineUrl,
+                        shouldHaveError: true,
+                        errorMessage: 'Invalid file',
+                    },
+                    {
+                        url: outOfBoundKMLUrl,
+                        shouldHaveError: true,
+                        errorMessage: 'out of projection bounds',
+                    },
+                ]
+
+                const { url, shouldHaveError, errorMessage } = errorData[index]
+
+                if (shouldHaveError) {
+                    cy.wrap($layer)
+                        .find('[data-cy^="button-has-error"]')
                         .should('be.visible')
-                        .contains('file not accessible')
-                } else if (index === 1) {
-                    cy.get(`[data-cy^="floating-button-has-error-${invalidFileOnlineUrl}"]`)
+                        .trigger('mouseover')
+
+                    cy.get(`[data-cy^="floating-button-has-error-${url}"]`)
                         .should('be.visible')
-                        .contains('Invalid file')
+                        .contains(errorMessage)
+
+                    // Mouseout to remove tooltip
+                    cy.wrap($layer).trigger('mouseout', { force: true })
```

Comment:
not sure this will really be removing the tooltip, as you are triggering the `mouseout` on a different element than the `mouseover`.
Might be a good idea to replicate the selector above and add the missing `.find('[data-cy^="button-has-error"]')` before triggering the event